src/components/icons/social/github.tsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import styled from 'styled-components';
 
+interface GithubProps {
```

Comment:
Thats only for the `GithubIconWrapper` right? Then the naming should be `GithubIconWrapperProps`. The interface should also be right above the implementation that you can see with one sight what the props are.

Reply:
It is not used in `GithubIconWrapper` but in the component `IconGithub` itself with `{...props}` 
(just like in `IconLinkedIn`)
Shall I write it extra with `opacity={props.opacity}`?